src/test/java/sirius/db/mongo/properties/MongoMultiLanguageStringEntity.java

```diff
@@ -30,24 +29,12 @@
     @NullAllowed
     private final MultiLanguageString multiLangTextWithFallback = new MultiLanguageString(true);
 
-    public static final Set<String> validLanguages = new HashSet<>(Arrays.asList("da",
-                                                                                 "nl",
-                                                                                 "en",
-                                                                                 "fi",
-                                                                                 "fr",
-                                                                                 "de",
-                                                                                 "hu",
-                                                                                 "it",
-                                                                                 "nb",
-                                                                                 "pt",
-                                                                                 "ro",
-                                                                                 "ru",
-                                                                                 "es",
-                                                                                 "sv",
-                                                                                 "tr"));
+    public static final List<String> validLanguages =
+            Arrays.asList("da", "nl", "en", "fi", "fr", "de", "hu", "it", "nb", "pt", "ro", "ru", "es", "sv", "tr");
```

Comment:
please check code format settings (mvn validate + refresh intellij) - either of both has invalid settings.

Reply:
this is correct code style, since the Statement does not overflow the new line

(checked multiple times after multiple mvn validate + clean + invalidate caches + ... 😉  )